Interesting build failure:

```
FAIL packages/block-editor/src/components/link-control/test/index.js
  ● Test suite failed to run
    Cannot find module '../../../../../data' from 'index.js'
```

I don't see it locally. But clearly related to the changes here.

I'll dig into it.

--- a/packages/block-editor/src/components/link-control/test/index.js
+++ b/packages/block-editor/src/components/link-control/test/index.js
@@ -17,16 +17,9 @@ import { fauxEntitySuggestions, fetchFauxEntitySuggestions } from './fixtures';
 
 const mockFetchSearchSuggestions = jest.fn();
 
-jest.mock( '@wordpress/data', () => {
-	return {
-		...require.requireActual( '../../../../../data' ),
-		useSelect() {
-			return {
-				fetchSearchSuggestions: mockFetchSearchSuggestions,
-			};
-		},
-	};
-} );
+jest.mock( '@wordpress/data/src/components/use-select', () => () => ( {
+	fetchSearchSuggestions: mockFetchSearchSuggestions,
+} ) );
 
 /**
  * Wait for next tick of event loop. This is required

